src/nanoemoji/nanoemoji.py

```diff
@@ -479,15 +483,16 @@ def write_svg_font_diff_build(
 
 
 def _input_files(font_config: FontConfig, master: MasterConfig) -> List[Path]:
-    if font_config.has_bitmaps:
-        input_files = [bitmap_dest(f) for f in master.sources]
-    elif font_config.has_picosvgs:
+    if font_config.has_picosvgs:
         input_files = [
             picosvg_dest(font_config.clip_to_viewbox, f) for f in master.sources
         ]
-    else:
-
+    elif font_config.has_untouchedsvgs:
         input_files = [abspath(f) for f in master.sources]
+    elif font_config.has_bitmaps:
+        input_files = [bitmap_dest(f) for f in master.sources]
+    else:
+        raise AssertionError(f"Unsupported color_format: '{font_config.color_format}'")
     return input_files
```

Comment:
I think maybe these should be just ifs, it's weird that has_picosvgs AND has_bitmaps can both be true but then we don't list the bitmaps as inputs.  You'd have to adjust how write_font receives input but ... I think that's more correct. And then you can delete the implicit deps business in write_static_font_build.